Update FormatLocalizedString limits wording (#109608)
Azure/azure-content: articles/active-directory-b2c/string-transformations.md

| OutputClaim | outputClaim | string | The claim that is produced after this claims transformation has been invoked. |

> [!NOTE]

> > There are no limits to the number of input claims that can be specified, but the maximum length of the formated string is 4000.

Reviewer: ```suggestion
> There is no limit to the number of input claims that you can specify, but the maximum length of the formatted string is 4000.
```
